### PR TITLE
Clean up re:invent takeover and add some use case callouts

### PR DESCRIPTION
**Pull request by zchase**
During re:Invent we added some sections to the `/aws` page and this PR removes those sections and replaces them with more generic feature callout sections.

### CODE REVIEW

File: layouts/partner/aws.html [hunk 1]
```diff
@@ -281,6 +208,115 @@ <h3 class="mt-12 mb-8 text-center">
...
+                    <a class="inline-flex items-center text-gray-700 hover:text-gray-800 font-bold text-xl" href="{{ relref . "/blog/automation-api" }}">
+                        <span class="inline-block w-6 text-orange-300 mr-3">
+                            <svg aria-hidden="true" focusable="false" data-prefix="fad" data-icon="sparkles" class="svg-inline--fa fa-sparkles fa-w-16" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><g class="fa-group"><path class="fa-secondary" fill="currentColor" d="M423.16 186.58L448 127l59.58-24.84a8 8 0 0 0 0-14.32L448 63 423.16 3.42a8 8 0 0 0-14.32 0L384 63l-59.58 24.84a8 8 0 0 0 0 14.32L384 127l24.84 59.58a8 8 0 0 0 14.32 0zm-14.32 136.84L384 383l-59.58 24.84a8 8 0 0 0 0 14.32L384 447l24.84 59.58a8 8 0 0 0 14.32 0L448 447l59.58-24.84a8 8 0 0 0 0-14.32L448 383l-24.84-59.58a8 8 0 0 0-14.32 0z" opacity="0.4"></path><path class="fa-primary" fill="currentColor" d="M384 254.64a16.06 16.06 0 0 0-8.84-14.33l-112.57-56.39-56.28-112.77c-5.44-10.87-23.19-10.87-28.62 0l-56.28 112.77L8.84 240.31a16 16 0 0 0 0 28.67l112.57 56.39 56.28 112.77a16 16 0 0 0 28.62 0l56.28-112.77L375.16 269a16.07 16.07 0 0 0 8.84-14.36z"></path></g></svg>
```
Comment (cnunciato) [inline layouts/partner/aws.html:227]: We've copied this around a bunch now, looks like. How about we pull it into a parameterized partial or something to reduce the duplication and make a more generally useful thing?

File: layouts/partner/aws.html [hunk 2]
```diff
@@ -281,6 +208,115 @@ <h3 class="mt-12 mb-8 text-center">
...
+                    well-architected best practices to make common infrastructure-as-code
+                    tasks in AWS easier and more secure. Secure and cost-conscious defaults are chosen
+                    so that simple programs automatically use best practices for the underlying
+                    infrastructure, enabling better productivity with confidence.
+                </p>
+
+                <p class="text-lg font-bold">Application and Infrastructure Together</p>
+                <p>
+                    Crosswalk enables you to blur the lines between application and infrastructure code enabling
+                    you to author an entire full stack application in one program. With support for inline Lambda
```
Comment (cnunciato) [inline layouts/partner/aws.html:265]:
```suggestion
                    you to author an entire full-stack application in one program. With support for inline Lambda
```

File: layouts/partner/aws.html [hunk 3]
```diff
@@ -281,6 +208,115 @@ <h3 class="mt-12 mb-8 text-center">
...
+                {{ if eq .Params.title "Building a Kubernetes Platform in Amazon EKS with Pulumi" }}
+                    {{ $reInventWorkshops = $reInventWorkshops | append . }}
+                {{ else if eq .Params.title "Deploying Microservices with Pulumi and AWS Lambda" }}
+                    {{ $reInventWorkshops = $reInventWorkshops | append . }}
+                {{ else if eq .Params.url_slug "introduction-to-pulumi" }}
+                    {{ $reInventWorkshops = $reInventWorkshops | append . }}
+                {{ end }}
+            {{ end }}
+
+
```
Comment (cnunciato) [inline layouts/partner/aws.html:303]:
```suggestion
```

File: layouts/partner/aws.html [hunk 4]
```diff
@@ -281,6 +208,115 @@ <h3 class="mt-12 mb-8 text-center">
...
+                        <span class="inline-block w-6 text-orange-300 mr-3">
+                            <svg aria-hidden="true" focusable="false" data-prefix="fad" data-icon="sparkles" class="svg-inline--fa fa-sparkles fa-w-16" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><g class="fa-group"><path class="fa-secondary" fill="currentColor" d="M423.16 186.58L448 127l59.58-24.84a8 8 0 0 0 0-14.32L448 63 423.16 3.42a8 8 0 0 0-14.32 0L384 63l-59.58 24.84a8 8 0 0 0 0 14.32L384 127l24.84 59.58a8 8 0 0 0 14.32 0zm-14.32 136.84L384 383l-59.58 24.84a8 8 0 0 0 0 14.32L384 447l24.84 59.58a8 8 0 0 0 14.32 0L448 447l59.58-24.84a8 8 0 0 0 0-14.32L448 383l-24.84-59.58a8 8 0 0 0-14.32 0z" opacity="0.4"></path><path class="fa-primary" fill="currentColor" d="M384 254.64a16.06 16.06 0 0 0-8.84-14.33l-112.57-56.39-56.28-112.77c-5.44-10.87-23.19-10.87-28.62 0l-56.28 112.77L8.84 240.31a16 16 0 0 0 0 28.67l112.57 56.39 56.28 112.77a16 16 0 0 0 28.62 0l56.28-112.77L375.16 269a16.07 16.07 0 0 0 8.84-14.36z"></path></g></svg>
+                        </span>
+                        <span class="mr-2 hover:mr-3 transition-all">
```
Comment (cnunciato) [inline layouts/partner/aws.html:274]: If you look around the code, you'll see that we almost always use even-numbered margins and padding. We should stick this wherever possible for visual consistency. For example, this is too narrow on both sides (it's outside this PR, but taken from this page):

image

These should be `p-4` or `m-4` or the like. If you could go through and make sure we're being consistent here, I'd really appreciate it; I can't really keep a close watch on all of these, so I've tried to be as consistent as possible in the code throughout the site, hoping we'd use that for reference.

File: layouts/partner/aws.html [hunk 5]
```diff
@@ -281,6 +208,115 @@ <h3 class="mt-12 mb-8 text-center">
...
+                        </span>
+                    </a>
+                </p>
+            </div>
+        </div>
+    </section>
+
+    <!-- Start AWS re:Invent takeover content -->
+    <section id="workshops" class="container mx-auto text-center mb-12">
+        <p class="text-3xl font-bold">JOIN A WORKSHOP</p>
```
Comment (cnunciato) [inline layouts/partner/aws.html:288]: Shouldn't this be a heading (e.g., `h3`)?

**Review by cnunciato — COMMENTED** [review]
Looks mostly good, but a couple of visual things we should clean up:

The code samples are missing horizontal margins on mobile:

image

Tile margins are uneven:

image

File: layouts/partner/aws.html [hunk 6]
```diff
@@ -281,6 +208,115 @@ <h3 class="mt-12 mb-8 text-center">
...
+    <!-- Automation API -->
+    <section id="automation-api" class="container mx-auto mb-12">
+        <div class="flex flex-col lg:flex-row">
+            <div class="w-full lg:w-5/12 px-8">
+                <p class="text-3xl font-bold inline-block">Enable Everyone To Do Everything</p>
+                <p>
+                    The Automation API gives you the ability to build bespoke deployment tools that allow
+                    application developers to self-serve their infrastructure, using codified best practices.
+                    No two applications or environments are the same and you can quickly hit edges with tools
+                    trying to be one-size-fits all solution. The Automation API enables you to write tooling that fits
```
[the remark this answers is not in the document]
Comment (zchase) [reply layouts/partner/aws.html:220]: 👍

File: layouts/partner/aws.html (hunk 2 above)
Comment (zchase) [reply layouts/partner/aws.html:265]: 👍

File: layouts/partner/aws.html (hunk 3 above)
Comment (zchase) [reply layouts/partner/aws.html:303]: 👍

File: layouts/partner/aws.html (hunk 5 above)
Comment (zchase) [reply layouts/partner/aws.html:288]: 👍

File: layouts/partner/aws.html (hunk 4 above)
Comment (zchase) [reply layouts/partner/aws.html:274]: 👍

File: layouts/partner/aws.html (hunk 1 above)
Comment (zchase) [reply layouts/partner/aws.html:227]: 👍

**Review by cnunciato — APPROVED** [review]
🤘 :shipit: 🤘

**Pull request merged**